app/models/speaker.py

```diff
@@ -59,6 +59,7 @@ def __init__(self,
                  heard_from=None,
                  sponsorship_required=None,
                  event_id=None,
+                 user_id=None,
                  user=None):
```

Comment:
@LuD1161 I'm saying about the line below. Since you are directly setting up `user_id` therefore there is no requirement of line 63.